src/System.Threading/tests/System.Threading.Tests.builds

```diff
@@ -9,7 +9,7 @@
     </Project>
     <Project Include="System.Threading.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TestTFMs>net46</TestTFMs>
+      <TestTFMs>net463</TestTFMs>
```

Comment:
Is it valid to set the TargetGroup to net46 or net463 in tests\*.builds like in src\*.builds, such that it builds against the desktop surface area?